Ryujinx.Ava/UI/Models/InMemoryLogTarget.cs

```diff
@@ -0,0 +1,78 @@
+using Avalonia.Media;
+using Ryujinx.Common.Logging;
+using System.Collections.ObjectModel;
+
+namespace Ryujinx.Ava.UI.Models
+{
+    public class InMemoryLogTarget : ILogTarget
+    {
+        public class Entry
+        {
+            public Color Color { get; set; }
+            public string Text { get; set; }
+        }
+
+        private static Color GetLogColor(LogLevel level) => level switch {
```

Comment:
```suggestion
        private static Color GetLogColor(LogLevel level)
        {
            return level switch
            {
```
Basically the same suggestion as the other PR.